app/mailers/notifier.rb

```diff
@@ -33,8 +33,17 @@ def account_update(args)
     send_nucore_mail args[:user].email, text("views.notifier.account_update.subject")
   end
 
+  # Notifies the specified recipient if an order is placed including a product
+  def product_order_notification(order_detail, recipient)
```

Comment:
I wouldn't mind this being a separate class (or on another mailer class). This notifier is pretty giant.

This is one of those places where it seems logical to follow existing apparent patterns, but we've decided we don't really like the pattern and have started to move away from it for new stuff, but the old stuff is still the old way.